runtime/appruntime/api/reqtrack.go

```diff
@@ -133,7 +133,7 @@ func (s *Server) finishRequest(output [][]byte, err error, httpStatus int) {
 	case model.AuthHandler:
 		req.Logger.Info().Dur("duration", dur).Msg("auth handler completed")
 	default:
-		if httpStatus != 0 {
+		if httpStatus != errs.HTTPStatus(err) {
```

Comment:
Now that it's always set, does it make sense to always log the http status?

Reply:
I believed it might be better to only log the status if it's different to the expected status (i.e how we currently do it)